airbyte-integrations/connectors/source-mixpanel/source_mixpanel/streams/export.py

```diff
@@ -123,7 +123,9 @@ def process_response(self, response: requests.Response, **kwargs) -> Iterable[Ma
             if item.get("time") and item["time"].isdigit():
                 item["time"] = datetime.fromtimestamp(int(item["time"])).isoformat()
 
-            yield item
+            # Even though the API is on DATE level, we still filter items compared to state cursor
+            if not "date" in stream_state or stream_state["date"] <= item["time"]:
```

Comment:
are both `stream_state["date"]` and `item["time"]` strings? Should you convert them to `datetime`?